src/AltinnCore/Designer/Controllers/DeployController.cs

```diff
@@ -147,7 +147,7 @@ public async Task<IActionResult> StartDeployment(string org, string service)
         [HttpGet]
         public async Task<IActionResult> FetchDeploymentStatus(string org, string service, string buildId)
         {
-            if (org == null || service == null || buildId == null)
+            if (org == null || service == null || buildId == null || buildId != "0")
```

Comment:
is there a possibility that buildId might be empty? so amy be use string.isnullorempty

Reply:
Edited to use string.isnullorempty for all input params instead